SU-SWS/stanford_profile
# D8CORE-1952: fixing card image width for a single on a no layout

# READY FOR REVIEW

# Summary
- Added a width to the single card image for a no layout. It works the same way for a full width layout page.

# Needed By (Date)
- ASAP

# Urgency
- high

# Steps to Test

1. pull in this branch
2. add a single card to a no layout page
3. Verify that the image fills the entire box in different widths

# Affected Projects or Products
- Multi-row

# Associated Issues and/or People
- D8CORE-1952
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
